app/views/batch_record_updates/index.html.erb

```diff
@@ -1 +1,20 @@
-<h1>Batch Record Updates</h1>
+<div class="home-page-section">
+  <h1>Batch Record Updates</h1>
+
+  <% if current_user %>
+  <h3>What would you like to do?</h3>
+  <div id="batch-record-menu">
+    <ul class="sub-title">
+      <li><%= link_to "Upload a batch", '#', data: { toggle: 'show', show: '#upload-batch-menu', hide: '#show-queue-menu #show-errors' } %></li>
+      <%= render partial: 'upload_batch_menu' %>
+      <li><%= link_to "Show queue", '#', data: { toggle: 'show', show: '#show-queue-menu', hide: '#upload-batch-menu #show-errors' } %></li>
+      <%= render partial: 'show_queue_menu' %>
+      <li><%= link_to "Show errors", '#', data: { toggle: 'show', show: '#show-errors', hide: '#show-queue-menu #upload-batch-menu' }%></li>
+      <%= render partial: 'show_errors' %>
+    </ul>
+  </div>
+  <%= render partial: 'shared/main_menu_button' %>
+  <% else %>
+  <p><%= link_to "Please login", get: 'home/index' %></p>
```

Comment:
Also, just as an FYI, this is going to create a link with the href `/home/index?get=home/index` which I don't believe is the desired behavior (but I could be wrong).